src/software/simulated_tests/ball_moves_from_rest.py

```diff
@@ -0,0 +1,51 @@
+import software.python_bindings as tbots_cpp
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class BallMovesFromRest(Validation):
+    """Checks if ball has moved 0.05m from initial position"""
+
+    def __init__(self, position):
+        self.initial_ball_position = position
+
+    def get_validation_status(self, world) -> ValidationStatus:
+        """CChecks if ball has moved 0.05m from initial position
+
+        :param world: The world msg to validate
+        :returns: FAILING if ball doesn't move according to RoboCup rules
+                  PASSING if ball moves according to RoboCup rules
+        """
+        validation_status = ValidationStatus.FAILING
+        current_ball_position = tbots_cpp.createPoint(
+            world.ball.current_state.global_position
+        )
+
+        if (self.initial_ball_position - current_ball_position).length() > 0.05:
```

Comment:
nit: we would prefer 0.05 to be a parameter we can set in the constructor: eg. `self.threshold`